ci-automation/ci-config.env

```diff
@@ -53,3 +53,12 @@ QEMU_BIOS="/usr/share/qemu/bios-256k.bin"
 # Published by vms.sh as part of the qemu vendor build.
 QEMU_UEFI_BIOS="flatcar_production_qemu_uefi_efi_code.fd"
 
+# -- Digital Ocean --
+
+# Use the "@ARCH@", "@CHANNEL@" and "@VERNUM@" placeholders. They will
+# be replaced.
+DO_IMAGE_URL="https://${BUILDCACHE_SERVER}/images/@ARCH@/@VERNUM@/flatcar_production_digitalocean_image.bin.bz2"
+DO_CONFIG_FILE='do-token.json'
```

Comment:
This name should be aligned with the credential parameter of the Jenkins job and use something like `="${NAME-"do-token.json"}"` if you want to provide a default here.

The other values `DO_REGION`, `DO_PARALLEL`, and `DO_MACHINE_SIZE` should also only fallback to defaults defined here and prefer what the Jenkins job sets up as env vars.

Reply:
Done.